CMakeLists.txt

```diff
@@ -1,100 +1,36 @@
 cmake_minimum_required( VERSION 3.0 )
-
 project ( SharpTNI VERSION 1.0 )
 
-set( CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR} ${CMAKE_MODULE_PATH} )
-
-set( sankoff_hdr
-  src/utils.h
-  src/basetree.h
-  src/sankoff.h
-  )
-
-set( sankoff_src
-  src/sankoffmain.cpp
-  src/utils.cpp
-  src/basetree.cpp
-  src/sankoff.cpp
-  )
-
-set( sample_sankoff_hdr
-  src/utils.h
-  src/basetree.h
-  src/sankoff.h
-  )
-
-set( sample_sankoff_src
-  src/samplesankoffmain.cpp
-  src/utils.cpp
-  src/basetree.cpp
-  src/sankoff.cpp
-  )
-
-set( dimacs_hdr
-  src/utils.h
-  src/basetree.h
-  src/dimacs.h
-  src/sankoff.h
-  )
-
-set( dimacs_src
-  src/dimacsmain.cpp
-  src/utils.cpp
-  src/basetree.cpp
-  src/sankoff.cpp
-  src/dimacs.cpp
-  )
-
-set( gamma_hdr
-  src/utils.h
-  src/basetree.h
-  )
-
-set( gamma_src
-  src/utils.cpp
-  src/basetree.cpp
-  src/gammamain.cpp
-  )
-
-set( LIBLEMON_ROOT "$ENV{HOME}/lemon" CACHE PATH "Additional search directory for lemon library" )
-
-set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11" )
+set( CMAKE_CXX_STANDARD 11 )
```

Comment:
This requires version CMake 3.1, so might need to be reverted?

Reply:
Thank you for pointing that out Alex!